test/kubemark/run-e2e-tests.sh

```diff
@@ -40,6 +38,6 @@ else
 	ARGS=$@
 fi
 
-go run ./hack/e2e.go -v --check_version_skew=false --test --test_args="--e2e-verify-service-account=false --dump-logs-on-failure=false ${ARGS}"
+#go run ./hack/e2e.go -v --check_version_skew=false --test --test_args="--e2e-verify-service-account=false --dump-logs-on-failure=false ${ARGS}"
```

Comment:
Fixed it. It's embarrassing how I unfailingly include this change always.
Maybe it's about time we fix this script to run the first command in case of being run inside a docker container and the second one otherwise. Would help save effort on local runs.
Do you think this would make sense?